package.json

```diff
@@ -11,13 +11,13 @@
     "prettier:test": "npx prettier -c .",
     "prestart": "next build",
     "start": "next start -p $PORT",
-    "tests": "jest && yarn test:e2e",
-    "test:unit": "node --expose-gc ./node_modules/.bin/jest --runInBand --logHeapUsage",
+    "tests": "NEXT_PUBLIC_ENV_NAME=test jest && yarn test:e2e",
+    "test:unit": "NEXT_PUBLIC_ENV_NAME=test node --expose-gc ./node_modules/.bin/jest --runInBand --logHeapUsage",
     "lint": "eslint . --max-warnings 0 --ext .js,.jsx && echo 'Lint complete.'",
-    "test:e2e": "PORT=5001 start-test 5001 cy:run",
-    "e2e:audit": "PORT=5001 start-test 5001 cy:run:audit",
-    "e2e:interactive": "PORT=5001 start-test 5001 cy:open",
-    "e2e:server": "PORT=5001 start-test 5001",
+    "test:e2e": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:run",
+    "e2e:audit": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:run:audit",
+    "e2e:interactive": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:open",
+    "e2e:server": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001",
```

Comment:
These will all be derived from `NODE_ENV` once we switch to dotenv-flow. There is an open PR for that but it will need updating.

Reply:
Do you need && before jest to execute the shell param store or is that not needed?

Reply:
I don't think it's needed - just checked and code executed from Jest can access this value.